sourcegraph/sourcegraph.proto

```diff
@@ -288,58 +289,115 @@ message RepoList {
 
 service RepoBadges {
 	// ListBadges lists the available badges for repo.
-	rpc ListBadges(RepoSpec) returns (BadgeList);
+	rpc ListBadges(RepoSpec) returns (BadgeList) {
+		option (google.api.http) = {
+			get: "/v1/repo_badges/list_badges"
```

Comment:
I'm not strongly in favor of either way, I just chose it because it seemed most other services did it that way. I wasn't aware GitHub API version was through headers, hmm.

> I think we'd want to avoid that situation in either case.

The version being in the URL makes it more preventable though. If we use `/foo/bar` once we can't redefine how it works ever again, but if we use `/v1/foo/bar` we can always use `/v2/foo/bar`.